src/main/java/bhs/devilbotz/commands/DriveSetDistancePID.java

```diff
@@ -4,20 +4,31 @@
 
 package bhs.devilbotz.commands;
 
+import bhs.devilbotz.Constants;
 import bhs.devilbotz.subsystems.DriveTrain;
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
+/**
+ * This command is a PID controller that drives the robot to a set distance. (Command does not check
+ * if robot is going straight)
+ */
 public class DriveSetDistancePID extends CommandBase {
   private DriveTrain drivetrain;
   private PIDController distance_pid;
   private double distance;
-
+  /**
+   * The constructor for the Drive Straight PID command.
+   *
+   * @param drivetrain The drive train subsystem.
+   * @param distance The distance (in meters) the robot needs to cover.
+   */
   public DriveSetDistancePID(DriveTrain drivetrain, double distance) {
     this.drivetrain = drivetrain;
     this.distance = distance;
-    distance_pid = new PIDController(0.25, 0, 0.1);
+    distance_pid =
```

Comment:
```suggestion
    distancePid =
```
```suggestion
  private PIDController distance_pid;
```

rename to distancePid (use camel case, not python formatting)